Doc/library/cmath.rst

```diff
@@ -89,11 +89,11 @@ Power and logarithmic functions
    logarithms.
 
 
-.. function:: log(x[, base])
+.. function:: log(x, base=None)
 
    Returns the logarithm of *x* to the given *base*. If the *base* is not
-   specified, returns the natural logarithm of *x*. There is one branch cut, from 0
-   along the negative real axis to -∞, continuous from above.
+   specified or is None, returns the natural logarithm of *x*. There is one
```

Comment:
In the interests of readability, I'd suggest reverting the change on this line - I think it's clear from context that "not specified" includes the case of deliberately specifying `None`.